.github/workflows/devcontainer.yml

```
@@ -0,0 +1,137 @@
+        id: check
+        run: |
+          if [ ! -f ".devcontainer/devcontainer.json" ]; then
+            echo "exists=false" >> $GITHUB_OUTPUT
+            echo "::warning::No .devcontainer/devcontainer.json found, skipping build"
+          else
+            echo "exists=true" >> $GITHUB_OUTPUT
+          fi
+      # repository owner to lowercase for Docker image naming, as devcontainers/ci does not safeguard
+      - name: Get lowercase repository owner
+        id: repo_owner
+        run: echo "owner_lc=$(echo '${{ github.repository_owner }}' | tr '[:upper:]' '[:lower:]')" >> $GITHUB_OUTPUT
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:1:90: Double quote to prevent globbing and word splitting

(shellcheck)